gulpfile.js

```diff
@@ -69,6 +70,37 @@ var webpackConfig = {
   plugins: gulpPlugins
 };
 
+gulp.task('test-unit', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js'
+    }, done).start();
```

Comment:
When running this on my local it seems to hang after running the tests. Any idea as to why?